avaframe/test3Num/simiSol.py

```diff
@@ -0,0 +1,340 @@
+""" This script calculates the similarity solution for a gliding avalanche on
+a inclined plane according to similarity solution from :
+Hutter, K., Siegel, M., Savage, S.B. et al.
+Two-dimensional spreading of a granular avalanche down an inclined plane
+Part I. theory. Acta Mechanica 100, 37–68 (1993).
+https://doi.org/10.1007/BF01176861
+"""
+
+# imports
+import numpy as np
+from scipy.integrate import ode
+import math
+import os
+
+# local imports
+from avaframe.in3Utils import cfgUtils
+import avaframe.com1DFAPy.com1DFA as com1DFA
+import avaframe.test3Num.simiSol as simiSol
+
+# Load configuration
+simiSolCfg = os.path.join('test3Num', 'simiSol_com1DFACfg.ini')
+cfg = cfgUtils.getModuleConfig(com1DFA, simiSolCfg)
```

Comment:
Same here
Why are you reading it in com1DFAPy folder and not it the test3Num folder?
I also noticed that you can put whatever module instead of com1DFA it still finds the ini file

Reply:
i am just using com1DFA to find the default file for this ini file and the file override option is then used by providing the path to the actual ini file I want to use